pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -126,6 +127,16 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.TopologyManagerPolicy != kubeletconfig.NoneTopologyManagerPolicy && !localFeatureGate.Enabled(features.TopologyManager) {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: topologyManagerPolicy %v requires feature gate TopologyManager", kc.TopologyManagerPolicy))
 	}
+
+	for _, nodeTaint := range kc.RegisterWithTaints {
+		if err := utiltaints.CheckTaintValidation(nodeTaint); err != nil {
+			allErrors = append(allErrors, fmt.Errorf("invalid taint: %v", nodeTaint))
+		}
+		if nodeTaint.TimeAdded != nil {
+			allErrors = append(allErrors, fmt.Errorf("taint TimeAdded is not nil"))
+		}
+	}
```

Comment:
This is missing a test... I know `CheckTaintValidation` appears to have some below.

We didn't have the coverage before, so I don't want to block this PR just on this one thing.

https://github.com/kubernetes/kubernetes/pull/105360 will make adding one much easier.

Reply:
I will follow this pr if it doesn't add `CheckTaintValidation` test function. Thanks for your remind.